@app/server/src/middleware/installSession.ts

```diff
@@ -75,7 +75,13 @@ export default (app: Express) => {
    * different authentication method such as bearer tokens.
    */
   const wrappedSessionMiddleware: RequestHandler = (req, res, next) => {
-    if (req.isSameOrigin) {
+    const origins = [];
+    if (process.env.SESSION_ALLOWED_ORIGINS) {
```

Comment:
Allow session cookies to be sent from origins other than the main client. Give the list of allowed origins.